### ISSUE TITLE
a11y(buttons): Low contrast on focused buttons

### ISSUE THREAD

**Issue by crisbeto** [angular/material2#1769]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
The buttons should have sufficient color contrast, both in regular mode and in high contrast mode.

#### What is the current behavior?
Currently, the button focus indication is a tint with 12% opacity, which is barely visible in regular mode and is completely invisible in high contrast mode. This makes it difficult to see whether a button has been focused.

CC @jelbourn
